src/misc/announcements.ts

```diff
@@ -32,3 +34,25 @@ setInterval(() => {
   index++;
   index = index % MESSAGES_LENGTH;
 }, INTERVAL_MINUTES * 60 * 1000);
+
+/**
+ * Broadcast the localized string to all players
+ */
+export function announceTranslation(translationKey: string, ...args: string[]) {
+  const players = Array.from(Bukkit.onlinePlayers);
```

Comment:
Copying to array is not ideal for performance, is there any reason why collection cannot be passed directly?